{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 1}, "tests": {"0": {"status": "ERROR", "message": "service = <selenium.webdriver.chrome.service.Service object at 0x000002776E0FB150>\noptions = 'C:\\\\Users\\\\islomdev\\\\.wdm\\\\drivers\\\\chromedriver\\\\win64\\\\115.0.5790.171\\\\chromedriver-win32/chromedriver.exe'\n\n    @staticmethod\n    def get_path(service: Service, options: BaseOptions) -> str:\n        path = service.path\n        try:\n>           path = SeleniumManager().driver_location(options) if path is None else path\n\nC:\\Users\\islomdev\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.common.selenium_manager.SeleniumManager object at 0x000002776E15DD90>\noptions = 'C:\\\\Users\\\\islomdev\\\\.wdm\\\\drivers\\\\chromedriver\\\\win64\\\\115.0.5790.171\\\\chromedriver-win32/chromedriver.exe'\n\n    def driver_location(self, options: BaseOptions) -> str:\n        \"\"\"\n        Determines the path of the correct driver.\n        :Args:\n         - browser: which browser to get the driver path for.\n        :Returns: The driver path to use\n        \"\"\"\n    \n>       browser = options.capabilities[\"browserName\"]\nE       AttributeError: 'str' object has no attribute 'capabilities'\n\nC:\\Users\\islomdev\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\common\\selenium_manager.py:71: AttributeError\n\nDuring handling of the above exception, another exception occurred:\n\n    @pytest.fixture()\n    def setup():\n>       driver = webdriver.Chrome(ChromeDriverManager().install())\n\nconftest.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nC:\\Users\\islomdev\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py:45: in __init__\n    super().__init__(\nC:\\Users\\islomdev\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py:51: in __init__\n    self.service.path = DriverFinder.get_path(self.service, options)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nservice = <selenium.webdriver.chrome.service.Service object at 0x000002776E0FB150>\noptions = 'C:\\\\Users\\\\islomdev\\\\.wdm\\\\drivers\\\\chromedriver\\\\win64\\\\115.0.5790.171\\\\chromedriver-win32/chromedriver.exe'\n\n    @staticmethod\n    def get_path(service: Service, options: BaseOptions) -> str:\n        path = service.path\n        try:\n            path = SeleniumManager().driver_location(options) if path is None else path\n        except Exception as err:\n>           msg = f\"Unable to obtain driver for {options.capabilities['browserName']} using Selenium Manager.\"\nE           AttributeError: 'str' object has no attribute 'capabilities'\n\nC:\\Users\\islomdev\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:40: AttributeError\n", "test_name": "test_customer_reg", "rerun": "0"}}, "suite_name": "test_001_CustomerRegistration.py"}}}, "date": "August 17, 2023", "start_time": 1692247803.8031952, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "0", "skip": "0", "error": "1", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}